modules/nested-tabs-html/assets/js/frontend/handlers/nested-tabs-html.js

```diff
@@ -414,112 +408,6 @@ export default class NestedTabsHtml extends Base {
 		return settings.classes.active;
 	}
 
-	getVisibleTabTitle( tabTitleFilter ) {
```

Comment:
This is old code used for a previous round of A11Y implementations, before we changed the Nested Elements markup.